core-bundle/contao/templates/modules/mod_navigation_accessible.html.twig

```diff
@@ -7,6 +7,15 @@
 %}
 
 {% block content %}
+    <script{{ attrs().setIfExists('nonce', csp_nonce('script-src')) }}>
+        let accessibleNavBreakpoint = parseInt(window.getComputedStyle(document.body).getPropertyValue('--nav-breakpoint'), 10) ?? 1024;
```

Comment:
```suggestion
        let accessibleNavBreakpoint = parseInt(window.getComputedStyle(document.body).getPropertyValue('--nav-breakpoint'), 10) || 1024;
```

Sorry, I was wrong before. Neither `getPropertyValue` nor `parseInt` will ever return `null`.

Reply:
We can go for this approach to still make it happen 😊 
<img width="482" height="339" alt="image" src="https://github.com/user-attachments/assets/afb006d9-74ef-4d64-8918-666e5930aa9b" />



```suggestion
        let navBreakpoint = parseInt(window.getComputedStyle(document.body).getPropertyValue('--nav-breakpoint'), 10)
        let accessibleNavBreakpoint = Number.isNaN(navBreakpoint ) ? 1024 : navBreakpoint;
```